flutter/Dockerfile

```diff
@@ -1,25 +1,26 @@
-FROM gitpod/workspace-full-vnc
+FROM gitpod/workspace-full
 
+USER root
```

Comment:
Nit: In general, we now prefer using only `USER gitpod` everywhere, and using `sudo` where root-rights are required.

This prevents Gitpod users from accidentally creating directories as `USER root`, and then not having access to them in their workspaces (without an easy way to fix it).

Reply:
That's right, let me change it.

Reply:
Done.